src/server/index.js

```diff
@@ -9,6 +9,7 @@ const jsonParser = bodyParser.json();
 
 const heartbeatHandler = require('./endpoints/heartbeatHandler');
 const loginHandler = require('./endpoints/loginHandler');
+const logoutHandler = require('./endpoints/logoutHandler');
```

Comment:
since we've split our files by endpoints, and the login and logout uses the same endpoint (/api/login) i would put these two functionality into the same file.